bindings/python/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(bindings)
 find_package(PCL COMPONENTS common REQUIRED)
 find_package(pybind11)
 
-pybind11_add_module(pcl /home/divyanshu/Projects/active/pcl/bindings/python/pybind11/common/impl/point_types.cpp)
+pybind11_add_module(pcl /home/divyanshu/Projects/active/pcl/bindings/python/pybind11/common/include/pcl/impl/point_types.cpp)
```

Comment:
There's `CMAKE_SOURCE_DIR`, `PROJECT_SOURCE_DIR`, `CMAKE_CURRENT_SOURCE_DIR`, etc. variables you can use. See [official docs](https://cmake.org/cmake/help/v3.5/manual/cmake-variables.7.html) for more details